### DISCUSSION TITLE
PrimeUI Update? Prime Components with VanillaJS (Pure JS)

### DISCUSSION THREAD

**Discussion by DesdeJorge** [primefaces/community#2653] [PrimeFaces] [answered]
Some years ago PrimeTek had a powerful library that allow to us use some prime components with JQuery without JSF, at that time was very common use everything like a "JQuery plugin" but. These days it's becoming more and more common the implementation of popular libraries with VanillaJS (Pure JS). I honestly think that if PrimeTek manages to create something like this, they could be a game changer.

It would be a giant step in the modernization of legacy applications, at least in their frontend. And Primetek, being a company that has the trust of many important companies in the market, could play a more than important role.

**Comment by melloware** [comment, marked as answer]
I believe PrimeTek is working on PrimeUI which is a pure VanillaJS using WebComponents framework.  I am pretty sure they announced it somewhere on one of Cagatay's videos.

see: https://github.com/primefaces/primeuix
